### PR TITLE
Add sca test file

### PR DESCRIPTION
**Pull request by Junefen**
Defined this test file to support sca testing of virt-who, initially developed 4 cases as below, will add more when get new requirements.

```
test_vw_report_mapping_with_sca_enabled
test_vw_host_to_guest_association
test_guest_entilement_with_sca_enabled
test_hypervisor_in_web
```

### CODE REVIEW

**Comment by Junefen** [comment]
Don't merge this pr, need some updates after define the api for stage candlepin.

File: tests/sca/test_virtwho_sca.py
Comment (Junefen) [inline tests/sca/test_virtwho_sca.py:17]: Maybe it's better to define a function ```sca_set()``` for the both satellite and stage using. Will perfect it in further prs

File: tests/sca/test_virtwho_sca.py
Comment (Junefen) [inline tests/sca/test_virtwho_sca.py:71]: This function need more efforts to research, will complete it when time is free.

**Review by hkx303 — APPROVED** [review]
ack, good jobs!

**Pull request merged**
